### DISCUSSION TITLE
Tutorial files for example 12.2 - Rainfall infiltration

### DISCUSSION THREAD

**Discussion by mikael-onsilva** [Anura3D/Anura3D_OpenSource#218] [General]
Does anyone has the files of the example 12.2 of the manual, the Rainfall infiltration? I'm trying to reproduce it here, it runs, but the results aren't the same as the manual and youtube video. 

If anyone could share the files, I would check what I'm missing.

Thanks in advance.

**Comment by mikael-onsilva** [comment]
I would add that when I try to run it for the first time, the _.PSF_1_ file it creates is empty and the run throws an error. Then if I copy the _.PSF_1_ file from the _.gid_ folder, the calculation runs.

**Comment by mikael-onsilva** [comment]
I've found out the reason for the error. Yet, the _.PSF_1_ created is still empty and if these files could be shared, it would be great.

**Reply by Air-shark** [reply]
I also had the same problem, please how did you solve it later.

**Comment by FrancescaCeccato** [comment]
I attch the file, hoping it is the same problem you are mentioning and that could be useful.
[PhreaticSurface.txt](https://github.com/user-attachments/files/15928625/PhreaticSurface.txt)
[RainfallInfiltrationCPS_001.txt](https://github.com/user-attachments/files/15928626/RainfallInfiltrationCPS_001.txt)
[RainfallInfiltrationGOM.txt](https://github.com/user-attachments/files/15928627/RainfallInfiltrationGOM.txt)

**Reply by FrancescaCeccato** [reply]
We've never used Anura3D for this application, I think it will require some adaptation. Thank you for your interest!

**Reply by Air-shark** [reply]
I was able to complete the result of the tutorial by replacing the GOM file, but I still don't know why my GOM file can't get the result.

**Reply by Air-shark** [reply]
[RainfallInfiltrationGOM-mycase.txt](https://github.com/user-attachments/files/17271901/RainfallInfiltrationGOM-mycase.txt)

**Reply by Aidamehr** [reply]
May I ask what that 13 at the very first row of PhreaticSurface means?

**Reply by FrancescaCeccato** [reply]
It is the number of points of the phreatic surface

**Reply by Aidamehr** [reply]
Thank you for your help

**Reply by Air-shark** [reply]
Dear FrancescaCeccato, I'm sorry to bother you. Recently, I was doing a simulation of the rapid decline of the water level of the dike(It is based on the tutorial manual, but there are 6 soil layers), but it reported an error. Do you have any solution? Error reported as follows:forrtl: severe (59): list-directed I/O syntax error, unit 100, file E:\Study\Anura3d\Zheng.A3D\Zheng.GOM

**Reply by FrancescaCeccato** [reply]
You have an error int he GOM file. Try to redraw the model.

**Reply by anijnini** [reply]
> GOM 文件有错误。尝试重绘模型。

Dear FrancescaCeccato! The GOM and CPS_001 files generated in Anura3D are both blank. Could you please tell me what the reason might be？

**Reply by FrancescaCeccato** [reply]
Thre is something going wrong in GiD. I don0t know what but you should check the problem type and rebuid the model.

**Reply by anijnini** [reply]
> 这是 GiD 中出了问题。我不知道是什么，但你应该检查问题类型并重新构建模型。
Thank you, dear Francesca Ceccato. I have another question: When I was using Anura3D 2022, I followed the example files in the user manual exactly as instructed, but the rainfall infiltration could not be realized. Specifically, when playing in ParaView, it did not slide. How can I solve this issue?

**Reply by FrancescaCeccato** [reply]
I don't know how to answer this question as I should check all the steps of your model. i can only say that we run several times all the examples of teh tutorial and they were working nicely.

**Reply by Air-shark** [reply]
[Zheng.GOM.txt](https://github.com/user-attachments/files/17573746/Zheng.GOM.txt)
In addition, I have attached my GOM file

**Comment by Air-shark** [comment]
Thank you very much for taking the time to answer my questions. I rebuilt the model and it's still the same problem. At present, I am replicating a paper(Pre- and post-failure behaviour of a dike after rapid drawdown of river level based on material point method), but when I completed the modeling, I could not perform the calculation, and I still could not find the error.I suspected that my hydraulic conditions or the diving surface had been set up wrong, but I was following the instructions in the tutorial manual, so I was confused.
[Zheng.gid.zip](https://github.com/user-attachments/files/17573584/Zheng.gid.zip)
I'm using Anura2023.

**Reply by Aidamehr** [reply]
Dear Air-Shark,
I wanted to ask whether you could come up with a solution, because, I faced many problems in my own model geometry. the tutorial models are working properly, but when I apply similar conditions to my model and geometry, I get an error.
I would be pleased to know whether you have an idea of what it stems from.
best regards

**Comment by Air-shark** [comment]
I got it. Check your liquid boundary conditions




---Original---
From: ***@***.***&gt;
Date: Wed, Dec 4, 2024 20:39 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Anura3D/Anura3D_OpenSource] Tutorial files for example 12.2 -Rainfall infiltration (Discussion #218)




 
Dear Air-Shark,
 I wanted to ask whether you could come up with a solution, because, I faced many problems in my own model geometry. the tutorial models are working properly, but when I apply similar conditions to my model and geometry, I get an error.
 I would be pleased to know whether you have an idea of what it stems from.
 best regards

**Reply by Aidamehr** [reply]
and may I ask if your result makes sense? because although I got an output, still the deformation and pore pressure doesn't look logical.

**Comment by Air-shark** [comment]
I match the results in the tutorial manual




---Original---
From: ***@***.***&gt;
Date: Wed, Dec 4, 2024 20:45 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Anura3D/Anura3D_OpenSource] Tutorial files for example 12.2 -Rainfall infiltration (Discussion #218)




 
and may I ask if your result makes sense? because although I got an output, still the deformation and pore pressure doesn't look logical.

**Reply by Aidamehr** [reply]
thanks for your answer.

**Comment by panf7422-art** [comment]
Does anyone has the files of the example 13.3 of the manual, the Rainfall infiltration?There is a display of a  'Norm of soild displacement' in the example,but I can't find it, Can you help me find out where it can be brought up?May I ask where I can find YouTube performance videos?
Thanks in advance.
